stories/Avatar/SmallOverlappingAvatars.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import AvatarIcon from '../../src/components/Avatar/AvatarIcon'
+import {Icon, LinkListItem} from '../../src'
+import AvatarGroupContainer from '../../src/components/Avatar/AvatarGroupContainer'
+import {LinkList, Dropdown,DropdownMenu,DropdownToggle } from '../../src'
+
+class SmallOverlappingAvatars extends React.Component {
```

Comment:
Refactor to a function component.
Make the state be handled by `UncontrolledDropdown` or knobs.